pandas/core/groupby.py

```diff
@@ -1475,9 +1475,169 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
+
         Return a new grouper with our resampler appended
+
+        Parameters
+        ----------
+        rule : str
+            Medio.
+        args
+            Hola.
+        kwargs
+            Chau.
+
+        Returns
+        -------
+        Grouper
+            Return a new grouper with our resampler appended
+
+        Examples
+        --------
+
+        Start by creating a series with 9 one minute timestamps.
+        >>> index = pd.date_range('1/1/2000', periods=9, freq='T')
+        >>> series = pd.Series(range(9), index=index)
+        >>> series
+        2000-01-01 00:00:00    0
+        2000-01-01 00:01:00    1
+        2000-01-01 00:02:00    2
+        2000-01-01 00:03:00    3
+        2000-01-01 00:04:00    4
+        2000-01-01 00:05:00    5
+        2000-01-01 00:06:00    6
+        2000-01-01 00:07:00    7
+        2000-01-01 00:08:00    8
+        Freq: T, dtype: int64
+
+        Downsample the series into 3 minute bins and sum the values
+        of the timestamps falling into a bin.
+        >>> series.resample('3T').sum()
```

Comment:
This method is `resample` on a groupby object, so the examples should also show that. See here for some explanation of this: https://pandas.pydata.org/pandas-docs/stable/groupby.html#new-syntax-to-window-and-resample-operations